corehq/apps/aggregate_ucrs/tests/data/table_definitions/basic_aggregate_definition.yml

```diff
@@ -0,0 +1,25 @@
+domain: agg
+table_id: pregnancies_with_forms
+display_name: Pregnancy Data With Form Aggregates
```

Comment:
I think it might not quite do that, but definitely should support it. I'll add a test and try to make that change soon. It currently requires the secondary table to have a column containing the ID of the primary table, but I think to link to a parent case you would need to be able to do the join using a different column on the primary table. Should hopefully be easy to add.

Reply:
(btw I think you found it but this works now)